Feature/adding angular to lib folder

islavi opened the pull request:
The following changes where done:
1. Move scripts from root folder (do not pollute the root folder) to utils folder: create-icon-map.js, build-demo.js.
2. Added scripts to build Angular (inside utils folder)
3. Update tsconfig.json
4. Updating index.js to include also Angular
5. Renaming Angular2 to Angular
6. Update the package.json:
   - Seperate the scripts according to functionality: build-common, build-angular, build-react, copy, publish
   - Removing "files" section (npm does not pack folders inside lib folder), and packing/publishing from lib folder using "npm run pub"
   - Updating the dependency list
7. delete "css" generated folder, and change the scripts to generate it inside lib folder.

islavi committed:
Merge pull request #4 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #5 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #6 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #7 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #8 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #9 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #10 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #11 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #12 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

stanvishne commented:
got this error when trying to import SdcUi from 'sdc-ui'
built file is called index.d.ts please fix. @islavi 

```
ERROR in ./~/sdc-ui/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./angular/index.ts in C:\myApps\locations\locations\node_modules\sdc-ui\lib
@ ./~/sdc-ui/lib/index.js 12:14-43
```

islavi committed:
Merge pull request #13 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

islavi committed:
Merge pull request #14 from onap-sdc/master

Update from upstream repo onap-sdc/sdc-ui

nomrik commented:
still getting 
![image](https://user-images.githubusercontent.com/20299155/28674004-cc00db3c-72ec-11e7-9873-a9af74ddc9df.png)

looks like you need to load the polyfill from https://www.npmjs.com/package/reflect-metadata somewhere?

islavi commented:
Added import for "reflect-metadata" in index.ts

islavi committed:
The following changes where done:

Move scripts from root folder (do not pollute the root folder) to utils folder: create-icon-map.js, build-demo.js.
Added scripts to build Angular (inside utils folder)
Update tsconfig.json
Updating index.js to include also Angular
Renaming Angular2 to Angular
Update the package.json:
Seperate the scripts according to functionality: build-common, build-angular, build-react, copy, publish
Removing "files" section (npm does not pack folders inside lib folder), and packing/publishing from lib folder using "npm run pub"
Updating the dependency list
delete "css" generated folder, and change the scripts to generate it inside lib folder.
